lib/plugins/game.js

```diff
@@ -90,4 +90,10 @@ function inject (bot, options) {
   bot._client.on(brandChannel, (serverBrand) => {
     bot.game.serverBrand = serverBrand
   })
+
+  bot._client.on('ping', (data) => {
```

Comment:
pretty sure this is already handled in nmp

Reply:
I cannot find where it is in node-minecraft-protocol.

Reply:
https://wiki.vg/Protocol#Ping_2
https://wiki.vg/Protocol#Pong
This is a different ping packet.

Reply:
"Unknown what this packet does just yet, not used by the Notchian server or client. Most likely added as a replacement to the removed window confirmation packet.

"

Reply:
if you have more information, please edit wiki.vg
otherwise remove this code change

Reply:
Edited wiki.vg also I gave the wrong link for Pong.
https://wiki.vg/Protocol#Pong_2
https://wiki.vg/Protocol#Ping_2